docs/questions/capitalQuiz.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../../css/main.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Capitalsquiz</title>
+</head>
+<body>
+    <main>
+        <div class="big-wrapper">
+            <header>
+                <div class="container">
+                    <div class="logo">
+                        <img src="../../assets/bee.png" alt="Logo">
+                        <h3>QuizBee</h3>
+                    </div>
+                    <div class="links">
+                        <ul>
+                           <li><a href="https://github.com/RahulSingh9131" target="blank">Github</a></li>
```

Comment:
I think think if you want to open the link in a new tab then the correct attribute should be target="_blank" instead of target = "blank"